doc/source/rllib/package_ref/models.rst

```diff
@@ -0,0 +1,27 @@
+.. _model-reference-docs:
+
+Model APIs
+==========
+
+ModelV2 API (rllib.env.models.modelv2.ModelV2)
+++++++++++++++++++++++++++++++++++++++++++++++
+
+All RLlib neural network models have to be provided as ModelV2 sub-classes.
+
+.. autoclass:: ray.rllib.models.modelv2.ModelV2
```

Comment:
btw, this isn't showing up properly:

> classray.rllib.models.modelv2.ModelV2(obs_space: <Mock name='mock.spaces.Space' id='140216445787536'>, action_space: <Mock name='mock.spaces.Space' id='140216445787536'>, num_outputs: int, model_config: dict, name: str, framework: str)[source]

Reply:
This is an existing problem (not introduced by this PR) with `gym.spaces.Space`, it seems. See e.g. my screenshot below. I'm not sure how to fix this. Sphinx replaces the `gym.spaces.Space` typehint with some `<Mock name='mock.Space' id='...'>`-tag ?? (could be because of the string "Space" having a special meaning for Sphinx?).